src/ducks/components/OneAppLoading.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react'
+import withBreakpoints from 'cozy-ui/react/helpers/withBreakpoints'
+
+export const Placeholder = ({ width, height, autoMargin, style }) => {
+  const widthStyle = Array.isArray(width)
+    ? `${Math.random() * (width[1] - width[0]) + width[0]}rem`
+    : width
+  if (style == undefined) style = {}
+  style = {
+    ...style,
+    width: widthStyle,
+    height
+  }
+  if (autoMargin) style.margin = 'auto'
+  return <span className="sto-sections-placeholder" style={style} />
+}
+
+export class OneAppLoading extends Component {
```

Comment:
Completely agree about the `<ApplicationPageLoading />` name.
Both solutions are good to me, do the condition before allow us to avoid a render and make the route render directly the loading interface.